src/app/modules/feedback/feedback.service.ts

```diff
@@ -151,3 +157,103 @@ export const hasNoPreviousFeedback = (
     }
     return okAsync(true as const)
   })
+
+/**
+ * Inserts given admin feedback to the database.
+ * @param userId the userId of the admin that provided the feedback
+ * @param rating the feedback rating to insert (0 for thumbs down, 1 for thumbs up)
+ * @param comment the feedback comment to insert if available
+ * @returns ok(IAdminFeedbackSchema) if successfully inserted
+ * @returns err(DatabaseError) on database error
+ */
+export const insertAdminFeedback = ({
```

Comment:
I'm thinking if it makes sense to separate functions that are related to `admin-feedback` as having its own service. It doesn't seem like there's much in common other than having the same term `feedback`.

Reply:
Refactored to its own module [eb93502](https://github.com/opengovsg/FormSG/pull/6454/commits/eb9350216c394b8ebc259da973a6ffc10ca288bc)